packages/subgraph/manifest/subgraph.placeholder.yaml

```diff
@@ -32,7 +32,7 @@ dataSources:
   {{/dataSources.PluginSetupProcessors}}
```

Comment:
while I can't comment on the exact line we have Entities inside this manifest that do not exist. We should replace them with the actual entities.